config/locales/en.yml

```diff
@@ -462,6 +462,9 @@ en:
       how_can_i_submit_my_event_answer: "<p>To submit your event, click on \"Submit Event\" and fill out all the details. Your personal information (name and email) will not be public and is only asked so that we can contact you.</p>"
       how_can_i_add_tags: "How can I add a new tag to my event?"
       how_can_i_add_tags_answer: "<p>We currently only allow site administrators to add new tags; if you'd like to tag your event with a tag that is currently not available from the list, please drop us a line at %{travis_email}. We will try to process your request as quickly as possible and add your chosen tag to your event submission.</p>"
+      why_provide_a_link: "Why do I have to provide a link when I manage application and selection myself?"
+      why_provide_a_link_short: "Why do I have to provide a link?"
+      why_provide_a_link_answer: "When you manage the application and selection process yourself it is important to show applicants (and your community) that you are serious about diversity and inclusion and you are trying to make an effort. A link to a form for the application or a dedicated page on your event's page explaining how you will choose from applicants and what you're looking for and why and what you want to achieve by offering diversity tickets goes a long way."
```

Comment:
I think this is a great answer but a bit lengthy so I would change it (slightly) to make it easier to read:
“When you manage the application and selection process yourself it's important to show applicants (and your community) that you are serious about diversity and inclusion and you are trying to make an effort. A link to a form for the application or a dedicated page on your event's page explaining how you will choose from applicants, what you're looking for and why, and what you want to achieve by offering diversity tickets goes a long way.”

Reply:
(I pretty much only added commas and used a contraction "it is" => "it's").